kubernetes/linux/main.sh

```diff
@@ -1065,8 +1065,8 @@ if [ ! -f /etc/cron.d/ci-agent ]; then
       echo "*/5 * * * * root /usr/sbin/logrotate -s /var/lib/logrotate/ci-agent-status /etc/logrotate.d/ci-agent >/dev/null 2>&1" >/etc/cron.d/ci-agent
 fi
 
-setGlobalEnvVar AZMON_RESOURCE_OPTIMIZATION_ENABLED "${AZMON_RESOURCE_OPTIMIZATION_ENABLED}"
```

Comment:
why AZMON_RESOURCE_OPTIMIZATION_ENABLED  being removed? are we not using both AZMON_RESOURCE_OPTIMIZATION_ENABLED  and AZMON_WINDOWS_FLUENT_BIT_ENABLED ?  We have bunch of places still we are using AZMON_RESOURCE_OPTIMIZATION_ENABLED  .

Reply:
AZMON_RESOURCE_OPTIMIZATION_ENABLED is for DS only   this is for controlling RS collection using fluentd thus using AZMON_WINDOWS_FLUENT_BIT_ENABLED